scripts/retrieve_databundle.py

```diff
@@ -28,22 +28,36 @@
 
 """
 
-import logging, os, tarfile
+import os
+import tarfile
 from _helpers import progress_retrieve
 
+import logging
 logger = logging.getLogger(__name__)
 
 if __name__ == "__main__":
 
+    logging.basicConfig(format=snakemake.config['logging_format'],
+                    level=snakemake.config['logging_level']
+                    # Logging to file doesn't work; collission with _helpers.progress_retrieve
+                    # handlers=[logging.FileHandler(snakemake.log[0]), logging.StreamHandler()],
+                    )
+
     if snakemake.config['tutorial']:
         url = "https://zenodo.org/record/3517921/files/pypsa-eur-tutorial-data-bundle.tar.xz"
     else:
         url = "https://zenodo.org/record/3517935/files/pypsa-eur-data-bundle.tar.xz"
 
     tarball_fn = "./bundle.tar.xz"
 
+    logger.info(f"Downloading databundle from '{url}'.")
     progress_retrieve(url, tarball_fn)
 
-    tarfile.open(tarball_fn).extractall('./data')
+    to_fn = "./data"
```

Comment:
A question about this here: Why doesn't the path break in windows? I thought path were separated by '/'. Perhaps saver should be to rather using "data"?

Reply:
Yep, that's not a good style there; it still works due to [this tricky Python feature](https://docs.python.org/3/library/os.html#os.altsep), but is not very stable.

I'll quickly fix it.